Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -52,6 +53,13 @@ class HistoryPanel: UIViewController, HomePanel {
         return button
     }()
 
+    func fetchSyncedDevicesCount() -> Success {
```

Comment:
Nit: Using 'fetch' implies it will return the devices count instead of a blank maybe. I would either rename this method to `updateSyncedDevicesCount` or have it return the count using `Deferred<Maybe<Int>>`.